integration-tests/chopsticks/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "e2e",
+  "version": "0.0.1",
+  "description": "chopsticks integration tests",
+  "main": "src/index.ts",
```

Comment:
We don't need this. Also, I would add a `"private"` property to make sure we don't publish this package.

Reply:
The private field is new for me :+1: [48626b5](https://github.com/KILTprotocol/kilt-node/pull/614/commits/48626b53ebcdb7e1a8c66170da8c678e479e55b3)